12-watchtower-API.md

```diff
@@ -22,227 +22,320 @@ The scope of this bolt does not include:
+* [Encryption Algorithms and Parameters](#encryption-algorithms-and-parameters)
+* [Payment Modes](#payment-modes)
+* [No compression of justice transaction](#no-compression-of-justice-transaction)
 
-  * [Transaction Locator and Encryption Key](#transaction-locator-and-encryption-key)
-  * [Encryption Algorithms and Parameters](#encryption-algorithms-and-parameters)
-  * [WatchTower API](#WatchTower-api)
-  * [Signed Receipt Fields](#signed-receipt-fields)
-  * [Payment Modes](#payment-modes)
-  * [Number of updates](#number-of-updates)
-  * [No compression of justice transaction](#no-compression-of-justice-transaction)
+## WatchTower discovery
+At this point we're leaving the client/server connection to be protocol agnostic. How the Lightning node finds the WatchTower or how the WatchTower announces their presence and services provided is not specified.
 
-## Connection establishment
-Connections between the client and the server can be long-lived or restarted for every single appointment.
+Therefore, we are assuming that the client and server are connected and that the client have learnt what Quality of Service (`qos`) the tower is offering. Moreover, we asume that the client has an authentication token `aut_token` to prove he's entitled to use the service if required.
 
-		+-------+                      +-------+
-		|   A   |--(1)--- wt_init ---->|   B   |
-		|       |<-(2)--- wt_init -----|       |
-		+-------+                      +-------+
+## WatchTower services
+
+### Basic Service
+The customer can hire the WatchTower to watch for breaches on the blockchain and relay a justice transaction on their behalf. The customer receives an acknowledgement when the WatchTower has accepted the job, but the hiring protocol does guarantee the transaction inclusion.
+
+### Quality of Service
+Quality of Service (`qos`) builds on top of the basic service provided by a tower and it's optionally provided. Different kinds of QoS can be offered by the tower.
+
+For now we are defining a single type of `qos`: `accountability`.
+
+#### `accountability`
+
+A WatchTower provides a signed receipt to the customer. This is considered reputational accountability as the customer has publicly verifiable cryptographic evidence the WatchTower was hired. The receipt can be used to prove the WatchTower did not relay the justice transaction on their behalf and/or request a refund.
+
+## Sending and receiving appointments
+
+Once the client is aware of the services provided by the server, the former can start sending appointments to the latter.
+
+		+-------+                                    +-------+
+		|   A   |--(1)---      appointment      ---->|   B   |
+		|       |<-(2)---   accepted/rejected   -----|       |
+		+-------+                                    +-------+
 		
 		- where node A is 'client' and node B is 'server'
 
-### The `wt_init` message
+### The `appointment` message
```

Comment:
There's no definition of what an appointment is. It should be explained that an appointment is a way that watch-tower is hired to do it's watching services.